modules/ROOT/pages/transaction-management.adoc

```diff
@@ -147,6 +147,10 @@ When running in a transactional scope, the entire process must run in the same t
 
 Any other router or scope (for example, Foreach) continues with the transaction and its behavior is not modified.
 
+=== Execution Engine Processing Type Selection
+
+Event processors provide hints to the runtime on which xref:execution-engine.adoc[Processing Type] should be used to perform their task. As transaction scopes are set to run on the same thread, the Processing Type hint will be ignored.
```

Comment:
Based that this doc is about "Transaction Managemenet" in the first phrase I would mention that is the "normal behaviour in non TX context" something like:
`When processing a flow outside of a transaction, Operations in the flow hints to the runtime on whcih....`
`When processing a flow inside of a transaction, the operations hin'ts are ignored and all the transaction is get's executed in the same flow`